### PR TITLE
Replace Hotkey library

### LINKED ISSUE #203: Get rid of `react-hotkeys`
If we want to update to React 18 in the future , the `react-hotkeys` library will likely errors that will be unfixable on our side (based on personal experience). Furthermore, `react-hotkeys` has been unmaintained for years so this will not get fixed on library side either.  We should therefore get rid of `react-hotkeys`, and replace it with a maintained library, e.g. `react-hotkeys-hook`.

### PR DESCRIPTION
**Pull request by Arnei**
Resolves #203 

Replaces react-hotkeys with react-hotkeys-hooks.
The latter is maintained, easier in use and already used in other Opencast frontend projects.

Also makes a small change to the hotkey cheat sheet, where hotkeys that can currently not be used are
displayed in a lower opacity.

### CODE REVIEW

**Comment by ziegenberg** [comment]
The browser console is complaining:

```
i18next::translator: missingKey en-GB translation HOTKEYS.KEYS.J j
i18next::translator: missingKey en-GB translation HOTKEYS.KEYS.M m
i18next::translator: missingKey en-GB translation HOTKEYS.KEYS.R r
i18next::translator: missingKey en-GB translation HOTKEYS.KEYS.H h
i18next::translator: missingKey en-GB translation HOTKEYS.KEYS.E e
i18next::translator: missingKey en-GB translation HOTKEYS.KEYS.S s
i18next::translator: missingKey en-GB translation HOTKEYS.KEYS.N n
```

Should we add the missing translations to en-US (see #406, why en-US and not en-GB)?

File: app/src/configs/hotkeysConfig.ts [hunk 1]
```diff
@@ -1,53 +1,45 @@
...
+		[key: string]: {
+			name: string,
+			description: string,
+			sequence: string[],
+		}
+	}
+}
+
+export const availableHotkeys: HotkeyMapType = {
 	general: {
```
[the remark this answers is not in the document]
Comment (Arnei) [reply app/src/configs/hotkeysConfig.ts:13]: Didn't think about translation strings at all, you should be right. Will delete those.

**Comment by Arnei** [comment]
> This probably also "solves" #73, right?

It will be "fixed" in the way you described in #73, yes.

**Review by ziegenberg — APPROVED** [review]

**Comment by ziegenberg** [comment]
Looks good to me 😊

**Pull request merged**